### PR TITLE
add amplication

### PR DESCRIPTION
**Pull request by souravjain540**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [X] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [X] I'm not adding a company from the blocklist
3. [X] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -150,27 +160,6 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
 - **Notes**: More details for the Hackathon prizes and Hacktoberfest details [are here](https://medusajs.com/blog/medusa-hackathon){:target="\_blank"}.
 
-#### **Meilisearch**
-
-- **Swag**: Reusable Bottles, Stickers (possibly? Awaiting updated swag for 2022.)
-- **Requirements**:
-    - Submit 1 or more pull requests to qualify as a contributor. Check out the [Contributing Guidelines](https://github.com/meilisearch/.github/blob/main/Hacktoberfest_2022_contributors_guidelines.md) for more details.
-- **How to sign up**: Join the [Slack community](https://www.notion.so/6cd1509802984122ab74745f637b694f) and the [#hacktoberfest channel](https://meilicommunity.slack.com/archives/C02G1H43J2X) to connect with Meilisearch employees and for any queries.
-- **Issues**: See the list of [hacktoberfest-labeled issues](https://github.com/search?q=org%3Ameilisearch+label%3Ahacktoberfest&state=open&type=Issues).
-- **Notes**: Check out the [blog post](https://blog.meilisearch.com/contribute-hacktoberfest-2022/) for more info.
```
Comment (crweiner) [inline docs/list.md:160]: Hi @souravjain540! 👋 Thank you for letting me know about Amplication! I'd love to get you all on the Swag List ASAP! Can you please rebase or re-add Meilisearch here? It seems like it's being accidentally deleted.

Thanks! 👍

File: docs/list.md [hunk 2]
```diff
@@ -50,6 +50,16 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **Issues**: List of [Hacktoberfest Labelled Issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc){:target="\_blank"}.
 - **Notes**: Read the [blog post](https://appwrite.io/hacktoberfest){:target="\_blank"} for more info.
 
+### **Amplication**
```
Comment (crweiner) [inline docs/list.md:53]: This just needs an extra space after the header to make the bots happy and so the md -> HTML build works correctly. 🤖 😄

**Review by crweiner — COMMENTED** [review]
Hi @souravjain540! Thanks again for your PR! Just a few little things to tweak, please, then we can get Amplication on the Swag List. 👍  Thanks for going above and beyond in celebrating Hacktoberfest this year!

**Comment by souravjain540** [comment]
@crweiner you can go ahead and merge now.

**Review by crweiner — APPROVED** [review]
Thank you for the updates, @souravjain540! Happy to get Amplication on the Swag List! Have a great Hacktoberfest and I hope to drive a good amount of meaningful PRs your way! 🎉

**Pull request merged**